### PR TITLE
Fixup package docs

### PR DESCRIPTION
**Pull request by olivroy**
Remove duplicated entry

### CODE REVIEW

File: DESCRIPTION
Comment (simonpcouch) [inline DESCRIPTION:1]: Could you revert the changes in this file?

File: R/broom-package.R [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+# Custom package description for the package help file.
+#' @description
+#' Convert statistical analysis objects from R into tidy tibbles,
+#' so that they can more easily be combined, reshaped and otherwise processed
+#' with tools like dplyr, tidyr and ggplot2. The package provides three S3
+#' generics: tidy, which summarizes a model's statistical findings such as
+#' coefficients of a regression; augment, which adds columns to the original
+#' data such as predictions, residuals and cluster assignments; and glance,
+#' which provides a one-row summary of model-level statistics.
+#'
```
Comment (simonpcouch) [inline R/broom-package.R:10]:
```suggestion
#' 
#' @aliases broom broom-package
#' 
```

So that folks can still `?broom` and that dead link will still work.

Comment (olivroy) [reply R/broom-package.R:10]: As you can see in the .Rd file, it still keeps the broom alias. So ?broom will work.

For keeping the broom link, I can add `@name broom`

**Comment by olivroy** [comment]
Comments addressed. The fixes won't break the link!

**Review by simonpcouch — APPROVED** [review]
Looking good. :)
